src/CMakeLists.txt

```diff
@@ -59,6 +59,7 @@ if( OPENSSL_VERSION LESS "1.0.0" )
     #ignore openssl code warnings
     set_source_files_properties( ${openssl_SRCS} PROPERTIES COMPILE_FLAGS ${OPENSSL_IGNORE} )
     list( APPEND digidocpp_SRCS ${openssl_SRCS} )
+    list( APPEND digidocpp_DEFS NO_CMS )
```

Comment:
I'd say that OPENSSL_NO_CMS would be the right one: https://android.googlesource.com/platform/external/openssl/+/eclair-release/crypto/opensslconf.h#52